sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -2,10 +2,22 @@
 
 ## 5.3.0b2 (Unreleased)
 
+This version of the client library defaults to the service API version `2022-10-01-preview`.
+
 ### Features Added
 
+- Added `begin_extract_summary` client method to perform extractive summarization on documents.
+- Added `begin_abstract_summary` client method to perform abstractive summarization on documents.
+
 ### Breaking Changes
 
+- Removed models `BaseResolution` and `BooleanResolution`.
```

Comment:
Is "BaseResolution" a base class that all derived resolutions extends from?

Reply:
Yes, but we don't really expose it and won't ever return it to the user and so architect feedback was to remove it.